index.bs

```diff
@@ -2572,8 +2576,17 @@ Council Deliberations</h5>
 	the [=W3C Council=] is considered to be convened
 	and can start deliberations.
 
-	Having reviewed the information gathered by the [=Team=],
-	the Council <em class=rfc2119>may</em> conduct additional research or analysis,
+	If a [=W3C Council=] has not yet been convened within 90 days of a Formal Objection
+	being [[#registering-objections|registered]],
+	the [=Chairs=] of the [=TAG=] and [=AB=] <em class=rfc2119>may</em> take independent action to ensure
+	that the [=dismissal=], [=renunciation=],
+	and [[#council-chairing|chair selection]] processes have been run.
+	Past the 90 days limit,
+	the [=Council=] is considered convened
```

Comment:
```suggestion
	and [[#council-chairing|chair selection]] processes have been run.
	Once those 90 days have elapsed,
	the [=Council=] is considered convened
```

Reply:
My first reaction was that your proposed phrasing was indeed better.

My second though is that it sounds like going past 90 days part of the normal course of action, but this clause is meant to be an exceptional fallback.